clusterloader2/cmd/clusterloader.go

```diff
@@ -290,6 +293,13 @@ func main() {
 		if err := prometheusController.SetUpPrometheusStack(); err != nil {
 			klog.Exitf("Error while setting up prometheus stack: %v", err)
 		}
+	} else if clusterLoaderConfig.PrometheusConfig.PreInstallServer {
+		if prometheusController, err = prometheus.NewController(&clusterLoaderConfig); err != nil {
```

Comment:
I'm just wondering about one thing. We heavily use prometheus-operator. In SetUpPrometheusStack we apply manifests that create PodMonitors/ServiceMonitors, you can check them here: https://github.com/kubernetes/perf-tests/tree/master/clusterloader2/pkg/prometheus/manifests/default
In your case though, we can't do it so some of the features of CL2 won't be working, unless you configure preinstalled prometheus manually. Is this something that is fine with your use case?